frontend/src/styles/buttons/ScrollToTopButton.module.css

```diff
@@ -0,0 +1,23 @@
+.ScrollToTopButton {
+    position: fixed;
+    display: inline-block;
+    right: 2%;
+    bottom: 20%;
+    z-index: 99;
+    cursor: pointer;
+    align-items: center;
+    justify-content: center;
+    object-fit: fill;
+  }
+  
+  .ScrollToTopButton .ScrollToTopButtonHover {
+    display: none;
+    left: 0;
+    position: absolute;
+    object-fit: fill;

```

Comment:
please consider when you add object-fit/any styles. "why am i putting this here, and what is it doing?"

Reply:
@heylookitsme Sorry if there is some unused or redundant styling, it might be some legacy CSS that I forgot to remove when trying to get the hover effect on the button to work properly originally.